source/scriptHandler.py

```diff
@@ -302,7 +302,7 @@ def executeScript(script, gesture):
 	finally:
 		_isScriptRunning = False
 		if resumeSayAllMode is not None:
-			sayAll.SayAllHandler.readText(resumeSayAllMode)
+			sayAll.SayAllHandler.readText(resumeSayAllMode, startedFromScript=None)
```

Comment:
Shouldn't this be

```suggestion
			sayAll.SayAllHandler.readText(resumeSayAllMode, startedFromScript=False)
```

Reply:
or is this meant to be `True`?

Reply:
It's `None` meaning that `sayAll.SayAllHandler.startedFromScript` should not be modified with this action. It should not be modified because this call is to resume an existing say all action, not to start it.

Should I make it somewhat more explicit? With a comment or otherwise?